packages/legend-query-builder/src/components/shared/BasicValueSpecificationEditor.tsx

```diff
@@ -190,6 +191,10 @@ const VariableExpressionParameterEditor = observer(
   },
 );
 
+const CustomSelectorInputInput = (
+  props: JSX.IntrinsicAttributes & InputProps,
+): ReactElement => <components.Input {...props} isHidden={false} />;
```

Comment:
why don't expose this `isHidden` as a config at the level of `CustomInputSelector`?